.devcontainer/post-start.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+set -eux
+
+# Add hosts
+sudo bash -c 'echo "127.0.0.1 dex" >> /etc/hosts'
+sudo bash -c 'echo "127.0.0.1 minio" >> /etc/hosts'
+sudo bash -c 'echo "127.0.0.1 postgres" >> /etc/hosts'
+sudo bash -c 'echo "127.0.0.1 mysql" >> /etc/hosts'
+sudo bash -c 'echo "127.0.0.1 azurite" >> /etc/hosts'
```

Comment:
what if these entries already exist in `/etc/hosts`? these may end up getting listed twice as a result. that won't break anything, but may want to check the file first

Reply:
Thanks for the review! I think it's impossible for the entries to already exist. Because when the container restarts it wipes the entries added by the script. But when we suppose that the entries already exist for some reason, that will add duplicate entries.
So I changed the code to make it more reliable by adding --add-host options as runArgs in devcontainer.json instead of using postStartCommand

Reply:
Oh good idea on `--add-host`!

If someone configured different networking or used a different client, there could be some overlap. But in general, just better to be safer if we can make the code purely additive without much effort